addons/pvr.iptvsimple/pvr.iptvsimple.json

```diff
@@ -11,9 +11,9 @@
     "build-options": {
         "no-debuginfo": true,
         "cflags": "-g0",
-        "cxxflags": "-g0"
+        "cxxflags": "-g0,$CXXFLAGS -Wp,-U_GLIBCXX_ASSERTIONS"
```

Comment:
Just pass `-Wp,-U_GLIBCXX_ASSERTIONS`. It will append it to `CXXFLAGS`. `$` variables aren't expanded.

Reply:
Thanks. Checking again, it seems like flathub already builds with `U_GLIBCXX_ASSERTIONS` so this isn't changing anything.